pkg/k8s/init.go

```diff
@@ -181,6 +182,11 @@ func WaitForNodeInformation(ctx context.Context, k8sGetter k8sGetter) error {
 
 		node.SetLabels(n.Labels)
 
+		if option.Config.NodeEncryptionOptOutLabels.Matches(k8sLabels.Set(n.Labels)) {
+			log.Info("Opting out on node-to-node encryption on this node")
```

Comment:
I wonder if it's worth adding a note on the origin of that decision? Simply something like:
```suggestion
			log.Info("Opting out on node-to-node encryption on this node as per user configuration")
```

Reply:
Good point, I'll add a reference to the `NodeEncryptionOptOutLabels` flag.